source/API_Reference/Event_Webhook/getting_started_event_webhook.md

```diff
@@ -0,0 +1,86 @@
+---
+seo:
+  title: Getting Started With the Event Webhook
+  description: Use RequestBin to get started or to troubleshoot your Event Webhook.
+  keywords: webhook, RequestBin, troubleshoot, event webhook
+title: Getting Started With the Event Webhook
+weight: 100
+layout: page
+navigation:
+  show: true
+---
+
+- [Getting started](#-Getting-started)
+- [Next steps](#-Next-steps)
+    - [Data storage](#-Data-storage)
+    - [Integrating](#-Integrating)
+
+SendGrid's Event Webhook will notify a URL of your choice via HTTP POST with information about events that occur as SendGrid processes your email. You can use this data to remove unsubscribes, react to spam reports, [determine unengaged recipients]({{site.blog_url}}/infer-engagement-with-the-event-api/), Identify bounced email addresses, or create advanced analytics of your email program. With Unique Arguments and Category parameters, you can insert dynamic data that will help build a sharp, clear image of your mailings.
+
+You should set up the Event Webhook if you want to keep track of more event data then we store for you. Due to the sheer volume of email we send, we can only store so much information. Your Email Activity Feed page holds records up to 500 events, and our internal database records only go back about 30 days. After that time passes, email event data is gone.
+
+{% info %}
+The Event Webhook is available to all accounts except [Legacy Lite]({{root_url}}/Classroom/Basics/Billing/legacy_lite_plan.html).
+{% endinfo %}
+
+{% anchor h2 %}
+Getting started
+{% endanchor %}
+
+*To get started with the Event Webhook:*
+
+1. Go to [RequestBin](https://requestb.in/). Select **Private (only viewable from this browser)**, select that you're not a robot, and then click **Create a RequestBin**.
```

Comment:
I would make "Select **Private (only viewable from this browser)**, select that you're not a robot, and then click **Create a RequestBin**." into a separate step.